examples/predictors/linear/2020-08-01_2020-08-04.csv

```diff
@@ -1,2411 +0,0 @@
-CountryName,RegionName,Date,PredictedDailyNewCases
```

Comment:
Delete old prediction files: generated predictions for each example are in a gitignore `predictions` folder.